CHANGELOG/CHANGELOG-1.6.md

```diff
@@ -0,0 +1,114 @@
+# Changelog 1.6
+
+## 1.6.0 - 2022-03-29
+
+The Egress feature is graduated from Alpha to Beta and is therefore enabled by default.
+
+### Added
+
+- Add support for the following capabilities for [Antrea IPAM] feature:
+  * Support pre-allocating continuous IPs for StatefulSet. ([#3281](https://github.com/antrea-io/antrea/pull/3281), [@annakhm])
+  * Support specifying VLAN for IPPool. Traffic from Pods whose IPPools are configured with a VLAN ID will be tagged when leaving the Node uplink. ([#3247](https://github.com/antrea-io/antrea/pull/3247), [@gran-vmv])
+- Add support for the following capabilities for [Antrea Multi-cluster] feature:
+  * Support Antrea ClusterNetworkPolicy replication in a ClusterSet. ([#3363](https://github.com/antrea-io/antrea/pull/3363), [@Dyanngg])
+  * Add `antctl mc` subcommand for Antrea Multi-cluster resources. ([#3341](https://github.com/antrea-io/antrea/pull/3341) [#3287](https://github.com/antrea-io/antrea/pull/3287), [@hjiajing] [@bangqipropel])
+- Add support for the following capabilities for [AntreaPolicy] feature:
+  * Add Node selector in Antrea-native policies to allow matching traffic originating from or destined for Nodes. ([#3038](https://github.com/antrea-io/antrea/pull/3038), [@wenqiq])
```

Comment:
s/originating from or destined for Nodes/originating from specific Nodes or destined to specific Nodes